symengine/flint_wrapper.h

```diff
@@ -476,6 +476,43 @@ class mpq_view_flint
     mpq_t m;
 };
 
+class fmpz_poly_factor_wrapper
+{
+private:
+    fmpz_poly_factor_t fac;
+
+public:
+    typedef fmpz_wrapper internal_coef_type;
+
+    fmpz_poly_factor_wrapper()
+    {
+        fmpz_poly_factor_init(fac);
+    }
+    fmpz_poly_factor_wrapper(const fmpz_poly_factor_wrapper &other)
+    {
+        fmpz_poly_factor_init(fac);
+        fmpz_poly_factor_set(fac, *other.get_fmpz_poly_factor_t());
+    }
+    fmpz_poly_factor_wrapper &operator=(const fmpz_poly_factor_wrapper &other)
+    {
+        fmpz_poly_factor_set(fac, *other.get_fmpz_poly_factor_t());
+        return *this;
+    }
+    ~fmpz_poly_factor_wrapper()
+    {
+        fmpz_poly_factor_clear(fac);
+    }
+
+    const fmpz_poly_factor_t *get_fmpz_poly_factor_t() const
```

Comment:
@isuruf, Earlier I wrote this pointer version on the lines of other getters like the one [here](https://github.com/ranjithkumar007/symengine/blob/6860d5c698a7c5e3ca53bac8eb1db709746a8bfe/symengine/flint_wrapper.h#L575). But anyways I have to dereference it later [here](https://github.com/symengine/symengine/pull/1274/files#diff-5d4d4c048c31a192019b52588f195ebbR161). So should this be changed to the one suggested above by @srajangarg?